src/include/kernel/abi.h

```diff
@@ -29,7 +29,7 @@
 
 /** \brief SOF ABI version major, minor and patch numbers */
 #define SOF_ABI_MAJOR 3
-#define SOF_ABI_MINOR 14
+#define SOF_ABI_MINOR 15
```

Comment:
@xiulipan This does not align with the classifier or the commit message. Can you double check this.

Reply:
@lgirdwood @xiulipan  D'oh, you are right. The kernel patch (not yet merged) puts cc_version as ABI16:
https://github.com/thesofproject/linux/pull/1890/commits/3552e9cace9b8a1be9c09202cc295557606de9de
So definitely commit message and patch are not in sync here.

This probably happened because FW PRs for ABI15 have been merged without bumping the version. E.g.
https://github.com/thesofproject/linux/pull/1890/commits/3552e9cace9b8a1be9c09202cc295557606de9de 
... was the last PR for ABI14, but it was merged with abi.h of ABI14.

I think we can probably still go ahead with this. The abi.h version needs to be changed to 16 when the last PR for ABI16 is merged. As @lyakh commented on the kernel side, if this is the norm to have multiple changes done in a single bump, we need to separate the commits that add functionality from commits that bump the version. I.e. we open ABIxx for development, have interface changes approved for this ABI level, and then we have a closing commit that freezes ABIxx. No binary FW releases should be done from master during this time.

Reply:
@kv2019i I see why I'm not finding the kernel PR's - @xiulipan can you split your PR into 3, one PR per ABI bump. That way we can align the with the FW on the classifier and follow. Once this is done and ABIs bumps aligned we can merge all at one go.

Reply:
@xiulipan ping ?

Reply:
@lgirdwood The three kernel pr is already there 
https://github.com/thesofproject/linux/pull/1947
https://github.com/thesofproject/linux/pull/1946
https://github.com/thesofproject/linux/pull/1890